sinks/external_test.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	kube_api "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	fuzz "github.com/google/gofuzz"
+	_ "k8s.io/heapster/api/v1/types"
```

Comment:
Not sure but without this the test fails. Probably a fuzzer checks what packages are initialized or so.